cmd/root.go

```diff
@@ -92,9 +92,9 @@ Complete documentation is available at https://appsody.dev`,
 		//Run: no run action for the root command
 	}
 
-	rootCmd.PersistentFlags().StringVar(&rootConfig.CfgFile, "config", "", "config file (default is $HOME/.appsody/.appsody.yaml)")
-	rootCmd.PersistentFlags().BoolVarP(&rootConfig.Verbose, "verbose", "v", false, "Turns on debug output and logging to a file in $HOME/.appsody/logs")
-	rootCmd.PersistentFlags().BoolVar(&rootConfig.Dryrun, "dryrun", false, "Turns on dry run mode")
+	rootCmd.PersistentFlags().StringVar(&rootConfig.CfgFile, "config", "", "Location of Appsody config files to use. (default '$HOME/.appsody/.appsody.yaml')")
```

Comment:
Does `--config` allow you to change the location and name of the default Appsody config file, or just the location (i.e. does the file have to be called .appsody.yaml)?

Reply:
Ok, so the expectation is that you make your own configuration file, called whatever you choose, and use `--config` to point at your own configuration file (so this definitely shouldn't say "files").